compiler/il/AliasSetInterface.hpp

```diff
@@ -274,7 +274,7 @@ class TR_AliasSetInterface {
       {
       TR::Compilation *comp = TR::comp();
       LexicalTimer t("removeAllAliases", comp->phaseTimer());
-      TR_ASSERT_FATAL(_AliasSetInterface == UseDefAliasSet || _AliasSetInterface ==  UseOnlyAliasSet, "failed: can only call removeAllAliases for UseOnly or useDef presently");
+      TR_STATIC_ASSERT(_AliasSetInterface == UseDefAliasSet || _AliasSetInterface ==  UseOnlyAliasSet, "failed: can only call setAliases_impl for UseOnly or useDef presently");
```

Comment:
Uhm, I don't think the assert message needs to change here. It was already correct.